<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 15px;
	background: #337AB7;
	}</string>
   </property>
   <widget class="QLabel" name="Choose">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 28px;
font-weight: bold;
color:#FFFFFF;
</string>
    </property>
    <property name="text">
     <string>Choose your theme:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>951</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Documents/AProg/gitlab/ToDo/src/Welcome_Window1/icons/wall.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Checkbox_Dark">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Checkbox.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="DarkDEal">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #949494;
font-size: 16px;
font-weight: bold;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style: normal;</string>
    </property>
    <property name="text">
     <string>Your Deal</string>
    </property>
   </widget>
   <widget class="QLabel" name="Checkbox_Light">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>checkbox_light.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="LightDeal">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #363636;
font-size: 16px;
font-weight: bold;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style: normal;</string>
    </property>
    <property name="text">
     <string>Your Deal</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Light_border_up">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>261</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFFFFF;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Light">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #363636;
font-size: 24px;
font-weight: bold;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style: normal;</string>
    </property>
    <property name="text">
     <string>Light</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="Dark_border_up">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>261</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1B1B1B;
borde-style: hidden;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="Dark">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #D7D7D7;
font-size: 24px;
font-weight: bold;
font-family: Calibri, Candara, Segoe, &quot;Segoe UI&quot;, Optima, Arial, sans-serif;
font-style: normal;</string>
    </property>
    <property name="text">
     <string>Dark</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>258</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton:clicked {
	background-color:#D7D7D7;
	border-style: hidden;
	}
QToolButton:hover {
	background-color: #D7D7D7;
	border-style: hidden;
	}
QToolButton {
	background-color: #D7D7D7;
	border-style: hidden;
	}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>258</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0D0D0D;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Letsgobutton_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color:#663366;
	color: white;
	font-size: 24px;
	font-weight:bold;
	font-family:Copperplate Gothic Bold, sans-serif;
	}
QPushButton:hover{
	background-color: #663385;
	color: #aaa;
	font-size: 24px;
	font-weight:bold;
	font-family:Copperplate Gothic Bold, sans-serif;
	}
QPushButton:pressed {
	background-color: #663350;
	color: white;
	font-size: 24px;
	font-weight:bold;
	font-family:Copperplate Gothic Bold, sans-serif;
	}</string>
    </property>
    <property name="text">
     <string>Create my list</string>
    </property>
   </widget>
   <zorder>Background</zorder>
   <zorder>toolButton</zorder>
   <zorder>toolButton_2</zorder>
   <zorder>Light_border_up</zorder>
   <zorder>Light</zorder>
   <zorder>Letsgobutton_2</zorder>
   <zorder>Dark_border_up</zorder>
   <zorder>Dark</zorder>
   <zorder>Choose</zorder>
   <zorder>LightDeal</zorder>
   <zorder>Checkbox_Light</zorder>
   <zorder>DarkDEal</zorder>
   <zorder>Checkbox_Dark</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>950</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
